docs/api/v1.18.md

```diff
@@ -256,8 +256,14 @@ Create a container
           should map to. A JSON object in the form
           `{ <port>/<protocol>: [{ "HostPort": "<port>" }] }`
           Take note that `port` is specified as a string and not an integer value.
-    -   **PublishAllPorts** - Allocates a random host port for all of a container's
+    -   **PublishAllPorts** - Allocates an ephemeral host port for all of a container's
           exposed ports. Specified as a boolean value.
+
+          Ports are de-allocated when the container stops and allocated when the container starts.
+          The allocated port might be changed when restarting the container.
+
+          On Linux the port is selected from ephemeral port range defined by
```

Comment:
Missing "the" -- "from the ephemeral port range"

Can we clarify the behavior on Windows Server?

Reply:
I looked in `./vendor/github.com/docker/libnetwork/portallocator` and didn't find anything about Windows.